app/src/main/java/ch/epfl/skysync/viewmodel/FinishedFlightsViewModel.kt

```diff
@@ -45,15 +45,24 @@ class FinishedFlightsViewModel(val repository: Repository, val userId: String) :
   private val _currentUser: MutableStateFlow<User?> = MutableStateFlow(null)
   private val isLoading = MutableStateFlow(false)
   private val _flightReports: MutableStateFlow<List<Report>?> = MutableStateFlow(null)
+  private val _flightReportsUsers: MutableStateFlow<List<User>?> = MutableStateFlow(null)
 
   val currentFlights = _currentFlights.asStateFlow()
   val currentUser = _currentUser.asStateFlow()
   val flightReports = _flightReports.asStateFlow()
+  val flightReportsUsers = _flightReportsUsers.asStateFlow()
 
   fun refresh() {
     refreshUserAndFlights()
   }
 
+  fun getUserWithId(id: String) {
+    viewModelScope.launch {
+      _currentUser.value = repository.userTable.get(id ?: UNSET_ID, onError = { onError(it) })
```

Comment:
`id ?: UNSET_ID` is unecessary and doesn't work anyway. Just use `id`

Reply:
fixed